In Elixir, you can use the `:logger` handler (`Sentry.LoggerHandler`) to automatically capture all logged exceptions, such as the ones from crashing processes. Alternatively, you can use `Sentry.capture_exception` to manually capture and report exceptions:

```elixir
try do
  1 / 0
rescue
  exception ->
    Sentry.capture_exception(exception, stacktrace: __STACKTRACE__)
end
```
